cf-agent/verify_users_pam.c

```diff
@@ -1610,20 +1622,6 @@ static bool DoModifyUser (const char *puser, const User *u, const struct passwd
         StringAppend(cmd, "\"", sizeof(cmd));
     }
 
-#ifndef __hpux
-    // HP-UX does not support -G with empty argument
-    if (CFUSR_CHECKBIT (changemap, i_groups) != 0)
-    {
-        /* Work around bug on SUSE. If secondary groups contain a group that is
```

Comment:
I checked this scenario on Suse Tumbleweed (2020-02-20) and did not see the problem mentioned here so think we may be safe without including this workaround. It seems the [original change](https://github.com/cfengine/core/commit/eb6ccf53afff33d50548bc8cd9d3e947b03eca45) is old enough (May 2014) that we don't have good tracking on what OS the problem was observed on.

Reply:
Issue is here: https://tracker.mender.io/browse/CFE-1468 but no mention of which version of SUSE so again I'm voting for "it works in the current version and so is good-to-go".

Reply:
Checked on hp-ux version ia64, B.11.23 and saw no problem with specifying `usermod -G ""` so think this may, like the SUSE workaround issue, be out-dated and a problem in some past version.